### PR TITLE
kc-mb-9798 - Update getting started designers

### PR DESCRIPTION
**Pull request by kennethchow-truss**
# Overview

These changes are related to [ticket 9798](https://dp3.atlassian.net/browse/MB-9798). It adds a few more lines of context for environment variables and adds the system requirements for the project

### CODE REVIEW

File: docs/frontend/setup/designers-guide-to-setting-up-app-locally.md [hunk 1]
```diff
@@ -66,15 +66,45 @@ For the sake of time let's try to install all of these dependencies at once. Som
...
 
-Most developers have AWS credentials that are able to keep these values synced and up to date.  In the scenario you don't have AWS permissions we will need to create a file with these pre-defined values filled in.  This file could become outdated as values change, get added or removed.  We should take care not to share this file publicly eventhough most of the values are not the same secrets we use elsewhere.
+Most developers have AWS credentials that are able to keep these values synced and up to date. In the scenario you don't have AWS permissions we will need to create a file with these pre-defined values filled in. This file could become outdated as values change, get added or removed. We should take care not to share this file publicly eventhough most of the values are not the same secrets we use elsewhere.
 
 Ask another developer with AWS credentials to send you the output of running:
 
-`DISABLE_AWS_VAULT_WRAPPER=1 AWS_REGION=us-gov-west-1 aws-vault exec transcom-gov-dev -- chamber list app-devlocal`
+```
+DISABLE_AWS_VAULT_WRAPPER=1 AWS_REGION=us-gov-west-1 aws-vault exec transcom-gov-dev -- chamber list app-devlocal
```
Comment (rogeruiz) [inline docs/frontend/setup/designers-guide-to-setting-up-app-locally.md:74]: I appreciate this call out, but I think for the OSS docs we want to point to Confluence documenting Chamber as it's more reliable and up to date with the system changes at the InfraSec level.

File: docs/frontend/setup/designers-guide-to-setting-up-app-locally.md [hunk 2]
```diff
@@ -66,15 +66,45 @@ For the sake of time let's try to install all of these dependencies at once. Som
...
+| aws_account_id       | 2       | 2020-01-01 12:00:00 | arn:aws-us-gov:sts::{arn_id}:assumed-role/infra/{arn_id} |
+| client_tls_cert      | 1       | 2020-01-01 12:00:00 | arn:aws-us-gov:sts::{arn_id}:assumed-role/infra/{arn_id} |
+| .....                | ....    | .....               | .....                                                    |
+| syncada_sftp_user_id | 1       | 2020-01-01 12:00:00 | arn:aws-us-gov:sts::{arn_id}:assumed-role/infra/{arn_id} |
+
+We need to create a new file called `.envrc.local` to paste these values into if it doesn't exist. Note that all files that start with a period are hidden so if somebody has already shared a file with you named `.envrc.local` it may not appear visible in the Finder like you expect.
+
+To create a new file named `.envrc.local` from inside the mymove folder in the Terminal run the command `touch .envrc.local && open -a TextEdit .envrc.local`. This should open a blank text file in TextEdit and you can paste and save the values.
+
+If you need to view this file again, go to the mymove folder on your Mac machine and use the command `command + shift + . (period)` to see hidden files.
```
Comment (rogeruiz) [inline docs/frontend/setup/designers-guide-to-setting-up-app-locally.md:90]: These are the docs we can reference to replace all this. https://dp3.atlassian.net/wiki/x/YoB6Sg 🔒

**Review by rogeruiz — CHANGES_REQUESTED** [review]
❌ This PR needs some changes only because we should move it over to Confluence. Good work on this @kennethchow-truss. I suggest keeping these changes light and pointing to Confluence to do the heavy lifting here.

File: docs/frontend/setup/designers-guide-to-setting-up-app-locally.md [hunk 3]
```diff
@@ -66,15 +66,45 @@ For the sake of time let's try to install all of these dependencies at once. Som
...
+
+If you need to view this file again, go to the mymove folder on your Mac machine and use the command `command + shift + . (period)` to see hidden files. 
+
+## System Requirements[#](https://transcom.github.io/mymove-docs/docs/frontend/setup/designers-guide-to-setting-up-app-locally/#defining-your-environment-variables-with-the-envrclocal)
+
+**macOS must be version 10.14 or newer**. That is, Mojave, Catalina, or Big Sur. We recommend upgrading to the latest version of macOS.**
+
+`Go` version 1.16.8
+
+`Node` v14.17.0
```
[the remark this answers is not in the document]
Comment (kennethchow-truss) [reply docs/frontend/setup/designers-guide-to-setting-up-app-locally.md]: Removed and changed to just link the page

File: docs/frontend/setup/designers-guide-to-setting-up-app-locally.md (hunk 1 above)
Comment (kennethchow-truss) [reply docs/frontend/setup/designers-guide-to-setting-up-app-locally.md]: Removed and added link to chambers page

File: docs/frontend/setup/designers-guide-to-setting-up-app-locally.md (hunk 2 above)
Comment (kennethchow-truss) [reply docs/frontend/setup/designers-guide-to-setting-up-app-locally.md]: Removed and added link to chambers page

**Review by rogeruiz — APPROVED** [review]
lgtm 🌈

Awesome work @kennethchow-truss. Some of the best docs are no docs. 🤣 Or rather, some of the best POAM docs are pointing to better docs. 👍🏽

**Pull request merged**